### PR TITLE
Adds a delay to table climbing and some flavor part two: electric boogaloo

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
:cl:
 * tweak: Added a delay before climbing tables to prevent powergaming. The Dwarf gene exists for a reason!

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
not a meme PR

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -357,19 +357,21 @@
 
 /obj/structure/table/MouseDropTo(atom/movable/O,mob/user,src_location,over_location,src_control,over_control,params)
 	if(O == user)
-		if(!ishigherbeing(user) || !Adjacent(user) || user.incapacitated() || user.lying) // Doesn't work if you're not dragging yourself, not a human, not in range or incapacitated
+		user.visible_message("span class='warning'>[user] tries to climb \the [src].</span>", "<span class='warning'>You try to climb \the [src].</span>")
+		if(do_after(user, src, 40))
+			if(!ishigherbeing(user) || !Adjacent(user) || user.incapacitated() || user.lying) // Doesn't work if you're not dragging yourself, not a human, not in range or incapacitated
```
Comment (ghost) [inline code/game/objects/structures/tables_racks.dm]: Why was this moved down? The visible message will be displayed if someone tries to drop themselves on a table from across the room.

File: code/game/objects/structures/tables_racks.dm [hunk 2]
```diff
@@ -359,17 +359,19 @@
...
+		if(do_after(user, src, 40))
+			var/mob/living/carbon/M = user
+			M.apply_damage(2, BRUTE, LIMB_HEAD, used_weapon = "[src]")
+			M.adjustBrainLoss(5)
+			M.Knockdown(1)
+			M.Stun(1)
+			if (prob(50))
+				playsound(M, 'sound/items/trayhit1.ogg', 50, 1)
+			else
+				playsound(M, 'sound/items/trayhit2.ogg', 50, 1)
```
Comment (beesil) [inline code/game/objects/structures/tables_racks.dm:372]:
```suggestion
			playsound(M, pick('sound/items/trayhit1.ogg','sound/items/trayhit1.ogg2'), 50, 1)
```

**Review by beesil — COMMENTED** [review]
might as well get rid of pointless if else statements while you're touching the code

**Comment by FlintBarrel** [comment]
I'm confused, i thought we didn't have table climbing?

**Comment by SonixApache** [comment]
@FlintBarrel 
You have to be on yellow intent, and it's only a small chance to succeed.

**Comment by jknpj** [comment]
rip in piss
